benchmarks/perf-tool/release-configs/faiss-hnsw/filtering/restrictive-filter/restrictive-filter-test.yml

```diff
@@ -0,0 +1,37 @@
+endpoint: [ENDPOINT]
+test_name: "Faiss HNSW Restrictive Filter Test"
```

Comment:
We need to add `port:` here and in other .yml files as well. Else, it will pick default port as 80 instead of 9200 for `localhost` and will fail

Reply:
making default value as 9200